admin_manual/configuration/occ_command.rst

```diff
@@ -11,6 +11,12 @@ interface. You can perform many common server operations with ``occ``::
 * Convert the ownCloud database from SQLite to a more performant DB
 * Query and change LDAP settings
 
+.. warning:: Always run the ``occ`` command as the HTTP user, because this
```

Comment:
HTTP user, httpd user, www-data user, xyz user

I think that highly depends on the used system/distro. :)